app/src/main/java/com/rkm/tasky/repository/model/Reminder.kt

```diff
@@ -0,0 +1,11 @@
+package com.rkm.tasky.repository.model
```

Comment:
I feel like this model is at the wrong place. At least I personally wouldn't expect a reminder model to be in a repository package.. Could it make more sense moving that in a dedicated models or reminder package?